apollo/__main__.py

```diff
@@ -206,14 +221,12 @@ def add_dzhigurda_arg(my_parser):
     add_spark_args(evalcc_parser, default_packages=[CASSANDRA_PACKAGE])
     add_cassandra_args(evalcc_parser)
 
-    # TODO: retable [.....] -> [.] [.] [.] [.] [.]
```

Comment:
This TODO still holds: it is about taking the extracted features and building the new hashtables with different parameters. Existing code does it in `hash` command. We probably just need the command to load the features from Cassandra and store them as batch ASDF files in the same format as `bags`.

Reply:
My bad, will put it back